Cargo.toml

```diff
@@ -85,8 +85,12 @@ threshold_crypto = { opt-level = 3 }
 
 [profile.ci]
 inherits = "dev"
-split-debuginfo = "packed"
-debug = 1
+# Minimize disk usage in CI to avoid running out of space
+incremental = false
+debug = 0
```

Comment:
But without it, if something panics/errors we don't get stacktraces, which is very handy when you're investigating some random flaky test.